testsuite/classlibrary/TestSimpleNumber.sc

```diff
@@ -0,0 +1,20 @@
+
+TestSimpleNumber : UnitTest {
+
+	test_snap {
+		var val = [ 0, 0.2, 0.4, 0.6, 0.8, 1, 1.2, 1.4, 1.6, 1.8, 2 ];
```

Comment:
just a note for future tests (although if you don't mind changing this one too, i'd appreciate it): you don't need this many test values to get full coverage. three or four should be enough — 0, a multiple of resolution, a value within tolerance, and a value outside tolerance would be a good set.

you're also missing test cases for when resolution is 0 or negative, tolerance is negative, and strength is 0 or negative. even if these are not intended use cases, it's good to pin them down so that if someone rewrites the method in the future, we will know what corner cases may have changed and can use that information responsibly.

i'd be happy to write those tests (testing is a very calming activity for me).

also, this file conflicts now with the TestSimpleNumber I added in my `asTimeString` PR. i think you should be fine just rebasing this on 3.9 (`git rebase 3.9 topic/quantize-function`) and then force-pushing.